### ISSUE TITLE
[BUG] A JavaScript error occurred in the main process: Cannot convert name to ASCII

### ISSUE THREAD

**Issue by bcutter** [mrvnklm/homeassistant-desktop#68]
v1.49, endless Error messages:

![grafik](https://user-images.githubusercontent.com/13799156/141647993-4e77d522-2a35-435a-8667-5be59e1dee7a.png)

Please assist what to do.

**Comment by mrvnklm** [comment]
Do you use some non english keyboard layout or anything else?

**Comment by bcutter** [comment]
> 

Yes, like I do on other Windows machines too without any issues. Don´t know what´s special about that one machine. Former versions were running just fine.
